app/gui/suggestion-database/src/entry.rs

```diff
@@ -432,7 +432,7 @@ impl Entry {
 
     fn code_with_static_this(&self) -> String {
         if let Some(self_type) = &self.self_type {
-            format!("{}{}{}", self_type.alias_name(), opr::predefined::ACCESS, self.name)
+            format!("{}{}{}", self_type.name(), opr::predefined::ACCESS, self.name)
```

Comment:
Unfortunately, it won't work with methods from Main modules of other projects. We need to check if the entry is actually from a currently opened project.

Reply:
Good catch. I've changed it. Do you want to have another look?